test/spec/test_spec.rb

```diff
@@ -0,0 +1,67 @@
+require_relative "spec_helper"
+
+describe "A PHP application" do
+
+  it "works with the getting started guide" do
+    Hatchet::Runner.new("php-getting-started").tap do |app|
```

Comment:
Please use `new_app_with_stack_and_platrepo` instead to ensure that the platform repository (which the buildpack pulls PHP etc from) the tests run against is the staging variant - `develop` and other branches then run against that, and `main` and tags run against the stable platform repo. That way, new PHP versions etc are tested before a merge to `main`.

Reply:
👍